frontend/public/components/monitoring/alerting.tsx

```diff
@@ -478,10 +478,10 @@ const Graph: React.FC<GraphProps> = ({
 };
 
 const tableSilenceClasses = [
-  classNames('col-sm-5', 'col-xs-8'),
-  classNames('col-md-2', 'col-sm-3', 'hidden-xs'),
-  classNames('col-md-3', 'col-sm-4'),
-  classNames('col-md-2', 'hidden-sm'),
+  'pf-m-width-50 pf-u-w-33-on-sm',
+  'pf-m-hidden pf-m-visible-on-sm pf-m-hidden-on-md pf-m-visible-on-lg',
```

Comment:
It seems like other instances the sizes descend from largest to smallest. Would be good to be consistent.

Reply:
I updated the sizes to consistent. Going from sm > md > lg > xl >2xl since PF rules apply to `*-on-{size}` and larger